pkg/services/ngalert/state/manager.go

```diff
@@ -418,6 +392,20 @@ func (st *Manager) staleResultsHandler(ctx context.Context, evaluatedAt time.Tim
 						previousState,
 					)
 				}
+
+				// If there is no resolved image for this rule then take one
+				if resolvedImage == nil {
+					image, err := takeImage(ctx, st.imageService, alertRule)
+					if err != nil {
+						logger.Warn("Failed to take an image",
+							"dashboard", alertRule.DashboardUID,
+							"panel", alertRule.PanelID,
+							"err", err)
```

Comment:
according to the new [guidelines ](https://github.com/grafana/grafana/blob/main/contribute/engineering/backend/instrumentation.md) we should use label `error` for errors